pages/data-design/avo-tracking-plan/events.mdx

```diff
@@ -181,13 +181,13 @@ analytics platform:
 
 #### Log Revenue
 
-It's recommended to add the Log Revenue action with events that represent a successful purchase, such as `Purchase Completed`
+Some analytics destinations have revenue tracking separate from event tracking. Add the *Log Revenue* action to indicate that the event represents a successful purchase and should be used to track revenue. When using Avo Codegen this will result in your event function receiving an additional _amount_ parameter that will then be passed on to the `revenue` function in your destination interface. Read more on it in [the Codegen docs](/reference/avo-codegen/destinations#destination-interface-callback-methods).
```

Comment:
```suggestion
Some analytics destinations have revenue tracking separate from event tracking. Add the *Log Revenue* action to indicate that the event represents a successful purchase and should be used to track revenue. When using Avo Codegen this will result in your event function receiving an additional _amount_ parameter that will then be passed on to the `logRevenue` function in your destination interface. Read more on it in [the Codegen docs](/reference/avo-codegen/destinations#destination-interface-callback-methods).
```

Reply:
Surprisingly, it's actually called `revenue`!